web-common/src/features/entity-management/resource-icon-mapping.ts

```diff
@@ -13,6 +13,7 @@ export const resourceIconMapping = {
   [ResourceKind.MetricsView]: DashboardIcon,
   [ResourceKind.API]: ApiIcon,
   [ResourceKind.Chart]: Chart,
+  [ResourceKind.Dashboard]: DashboardIcon, // TODO: show this be different?
```

Comment:
Yes, custom dashboards should use the bottom icon here:

![image](https://github.com/rilldata/rill/assets/14206386/e9b06ceb-ff18-4e79-9ed2-ed404f07a095)

Reply:
Will update once #4531 is merged